resources/wikia/modules/cookiePolicy.js

```diff
@@ -0,0 +1,55 @@
+/* global GlobalNotification */
+require(['wikia.cookies', 'wikia.geo'], function (cookies, geo) {
+	'use strict';
+
+	/**
+	 * Initialize JS for whether or not to show a banner notifying European Union users that we use cookies
+	 * on our site, per EU law. Users can dismiss the notification and will not be shown again, unless
+	 * they clear their cookies.
+	 */
+	function initCookieNotification() {
+		if (shouldShowBanner()) {
+			showBanner();
+		}
+	}
+
+	/**
+	 * Check if user is from the EU and hasn't seen the banner yet
+	 * @returns {boolean}
+	 */
+	function shouldShowBanner() {
+		var fromEU = geo.getContinentCode() === 'EU';
+
+		return (fromEU && !hasSeenBanner());
+	}
+
+	/**
+	 * Display the cookie policy message to the user
+	 */
+	function showBanner() {
+		var message = $.msg('oasis-eu-cookie-policy-notification-message') +
+			'<a href="http://www.wikia.com/Privacy_Policy#Cookies" target="_blank">' +
+			$.msg('oasis-eu-cookie-policy-notification-link') +
+			'</a>';
+
+		GlobalNotification.show(message, 'confirm');
```

Comment:
Our current implementation of ```GlobalNotification``` doesn't allow to display more than one message at the same time - new notification hides the old one. This could be in conflict with WikiaInYourLang extension: https://github.com/Wikia/app/blob/dev/extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js - which could display notification for first time users.